src/rust/rust-proto/src/graplinc/grapl/api/graph_query_proxy/v1beta1/messages.rs

```diff
@@ -0,0 +1,138 @@
+// Re-export the Response types
```

Comment:
## These 4 structs are exactly the same as the ones in `graph_query/messages.rs` except for the removal of `tenant_id` in the requests.

(technically, I could have reused the Response structs and just impld the TryFrom stuff for this proto instead of the other one; happy to reconsider that if people would prefer it?)